setup.py

```diff
@@ -10,24 +12,17 @@
     VERSION = re.search(r"__version__ = \"(.*?)\"", fh.read()).group(1)
 
 
-# the binary components of this library are pre-built
-# so setuptools can't tell our wheel should have a platform tag
-# therefore we use a custom bdist_wheel command to force it
-from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
-
-
 class bdist_wheel(_bdist_wheel):
     def finalize_options(self):
+        self.plat_name = get_platform()  # force a platform tag
         _bdist_wheel.finalize_options(self)
-        self.root_is_pure = False
 
 
 setup(
     name=NAME,
     version=VERSION,
     packages=find_packages(exclude=["tests", "tests.*", "examples", "examples.*"]),
     package_data={f"{NAME}.resources": ["*.dll", "*.so", "*.dylib", "*.h", "*.idl"]},
-    python_requires=">=3.6.0",
```

Comment:
don't we still need this? Or does it prevent us from creating the wheels we want. We can also add trove classifiers to specify the supported versions.

Reply:
I thought that it was an issue, but when I tried it again just now it worked just fine. So I restored it. Good call!